man/fi_endpoint.3.md

```diff
@@ -425,14 +427,14 @@ struct fi_info.  The following control commands and arguments may be
 assigned to an endpoint.
 
 **FI_GETOPSFLAG -- uint64_t *flags**
-: Used to retrieve the current value of flags associated with data
+: Used to retrieve the current value of flags associated with the data
```

Comment:
This should also state that FI_TRANSMIT/FI_SEND or FI_RECV must be provided as input, with the corresponding op_flags returned as output.